MetaMask/metamask-mobile
# chore: Typescript type coverage util  lib folders

## **Description**

Increased the Typescript coverage across the utils files.

## **Related issues**

Fixes: https://github.com/orgs/MetaMask/projects/60/views/16?pane=issue&itemId=69417476

## **Manual testing steps**

1. Go to `tsconfig.lint.json`
2. Uncomment the `// "app/util/**/*",` line
3. `yarn lint:tsc`
4. See if any errors show up from the `app/component` files

## **Screenshots/Recordings**



### **Before**

Typescript error present in test files listed in the ticket summary.

### **After**

No errors from those files

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
